entrypoint.sh

```diff
@@ -9,21 +9,21 @@ echo "$SIGUL_CONF" > /etc/sigul/client.conf
 echo "$SIGUL_PASS" > $SIGUL_PASS_FILE
 
 # Extract sigul config
-cd $HOME
+cd "$HOME" || exit
```

Comment:
Why exit if we can't find HOME or GITHUB_WORKSPACE? If these env vars don't exist, it represents a problem, and I don't think we want it to exit quietly.

Reply:
I could exit 1 instead.... I was just cleaning up because of shellcheck